extensions/blocks/story/player/style.scss

```diff
@@ -0,0 +1,377 @@
+@import '../../../shared/styles/gutenberg-base-styles.scss';
+@import '../../../shared/styles/jetpack-variables.scss';
+
+
+.wp-story-container {
+	height: 500px;
+	margin-left: auto;
+	margin-right: auto;
+	position: relative;
+	list-style: none;
+	padding: 0;
+	z-index: 1;
+	background-color: #0e1112;
+
+	&.wp-story-initialized {
+		opacity: 1;
+	}
+
+	.wp-story-clickable {
+		cursor: pointer;
+	}
+
+	// High specifity to override theme styles
+	.wp-story-wrapper,
+	.wp-story-slide {
+		padding: 0;
+		margin: 0;
+		line-height: normal;
+		list-style-type: none;
+	}
+
+	.wp-story-wrapper {
+		display: block;
+		position: absolute;
+		height: auto;
+		bottom: 0;
+		top: 0;
+		left: 0;
+		right: 0;
+		z-index: -1;
+	}
+
+	.wp-story-slide {
+		display: flex;
+		height: 100%;
+		width: 100%;
+		figure {
+			align-items: center;
+			display: flex;
+			height: 100%;
+			width: 100%;
+			justify-content: center;
+			margin: 0;
+			position: relative;
+
+			video {
+				width: 100%;
+			}
+		}
+	}
+
+	.wp-story-image, .wp-story-video {
+		display: block;
+		height: auto;
+		width: auto;
+		max-height: 100%;
+		max-width: 100%;
+		margin: 0;
+		border: 0;
+	}
+
+	.wp-story-meta, .wp-story-controls {
+		display: none;
+	}
+
+	.wp-story-button-fullscreen-prev,
+	.wp-story-button-fullscreen-next,
+	.wp-story-button-fullscreen-pause,
+	.wp-story-button-fullscreen-play {
+		background-color: rgba( 0, 0, 0, 0.5 );
+		background-position: center;
+		background-repeat: no-repeat;
+		background-size: 24px;
+		border: 0;
+		border-radius: 4px;
+		box-shadow: none;
+		height: 48px;
+		margin: -24px 0 0;
+		padding: 0;
+		transition: background-color 250ms;
+		width: 48px;
+
+		&:focus,
+		&:hover {
+			background-color: rgba( 0, 0, 0, 0.75 );
+		}
+
+		&:focus {
+			outline: thin dotted $white;
+			outline-offset: -4px;
+		}
+	}
+
+	.amp-carousel-button {
+		margin: 0;
+	}
+
+	&.player-container-rtl .player-button-prev.player-button-white,
+	&.player-container-rtl .wp-block-jetpack-story_button-prev,
+	.player-button-next.player-button-white,
+	.wp-block-jetpack-story_button-next {
+		background-image: url( "data:image/svg+xml,%3Csvg%20xmlns='http://www.w3.org/2000/svg'%20width='24'%20height='24'%20viewBox='0%200%2024%2024'%3E%3Cpath%20d='M5.88%204.12L13.76%2012l-7.88%207.88L8%2022l10-10L8%202z'%20fill='white'/%3E%3Cpath%20fill='none'%20d='M0 0h24v24H0z'/%3E%3C/svg%3E" );
```

Comment:
Hm, don't we have any way to inline SVGs in build-pipeline in Sass? 🤔 Would be much easier to maintain this if it would be in separate file.

Reply:
This code is outdated as well, sorry I need to clean up. All the icons I use for the player are actually encoded into a font that's inlined into the css here https://github.com/Automattic/jetpack/pull/16302/files/c3dd77328091b2e3d6da6bd920a01c6fa00686cc#diff-4c86797267f3a6de9c98fb77dd3ddc95R10

Reply:
Yup, I noticed the inspiration. :-) The slideshow was written some time ago (it also uses Classes vs hooks in functional component) so use your judgement.

Tiled gallery is another one where Slider originally took inspiration from, and Tiled gallery was largely copy of core gallery block. So I'd prolly check core gallery to make sure you have the greatest and latest. :-D

Reply:
> it also uses Classes vs hooks in functional component) so use your judgement.

Since you mention that. For new components / blocks, we now recommend that folks use function components, so I'd recommend going with that here as well.